msvc/scripts/csproj.tmpl

```diff
@@ -9,6 +9,9 @@
     <ProjectGuid>@PROJECTGUID@</ProjectGuid>
     <OutputType>@OUTPUTTYPE@</OutputType>
     <NoWarn>@DISABLEDWARNINGS@</NoWarn>
+    <HostPlatform Condition=" '$(HostPlatform)' == '' and '$(OS)' == 'Windows_NT'">win32</HostPlatform>

```

Comment:
Right now in the OutputPath (the property is written by the generator). In the future we can use it to select different files for different platforms.